card/mod/03_machines/lib/stylesheets/functional.scss

```diff
@@ -150,9 +150,9 @@ fieldset {
 }
 
 
-.navbox-form {
-  display: inline
-}
+//.navbox-form {
+//  display: inline
+//}
```

Comment:
I'm going to have a look at the css organization and (potentially) propose and work on re-arrangements.  But that can happen after merging.